packages/input-phone/src/PhoneInput.hooks.js

```diff
@@ -0,0 +1,108 @@
+import { useRef, useEffect, useMemo, useState, useCallback } from "react";
+import Fuse from "fuse.js";
+
+const options = {
+  shouldSort: true,
+  keys: ["name", "nativeNames", "altSpellings", "flag", "callingCode"],
+};
+
+export const useCountrySearch = (data, query, priority) => {
+  const fuse = useRef(new Fuse(data, options));
+
+  const allCountries = useMemo(() => {
+    const codes = priority?.split(",");
```

Comment:
I guess we should make sure this piece of code will never break either by wrapping it with a try catch block or by checking if `priority` is a string

Nothing presents user from passing an array, by mistake (or not lol), then it would break..